NOTES.md

```diff
@@ -0,0 +1,42 @@
+# Notes
+
+## Development Setup
+
+### Install Solana
+
+https://docs.solanalabs.com/cli/install
+
+- `sh -c "$(curl -sSfL https://release.solana.com/v1.18.4/install)"`
```

Comment:
I think `1.18.11` is out

Reply:
I just copied what was in the other README.md with the project and moved it here... thinking that the dir/files `sage-player-profile` will be deleted... if you go here still says v1.18.4 https://docs.solanalabs.com/cli/install

Reply:
I think the docs are not super actively maintained. The most recent `1.18` version will hit mainnet in a couple of days (possible fix for the congested network). I think we can have our docs up to date already.

Reply:
Ill merge. We can update the docs later.